sky/skylet/job_lib.py

```diff
@@ -187,6 +191,77 @@ def colored_str(self) -> str:
         return f'{color}{self.value}{colorama.Style.RESET_ALL}'
 
 
+class JobExitCode(enum.IntEnum):
+    """Job exit code enum.
+
+    These codes are used as return codes for job-related operations and as
+    process exit codes to indicate job status.
+    """
+
+    SUCCESS = 0
```

Comment:
Just to be aligned with our other enum keys?
```suggestion
    SUCCEEDED = 0
```